amqp/json_schemas/operation.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
oh just noticed something that will be difficult to present with schema, like that for example `mandatory` is allowed only if the operation of the binding is `publish` 😞

Reply:
It's possible to do that by `if/then/else` but we must infer type from specification schema, so only way to do that is probably custom validator in parser.

Reply:
yeah, let's not overcomplicate, this is not a blocker for this PR for sure